src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
```

Comment:
[CODE STYLE] - For consistency and adhering to the code style, use the assignment operator for multiplication as well. Replace `return a * b;` with `a *= b; return a;`.